### PR TITLE
Removed the duplicated masks for  DL1 parameters & images

### PR DESCRIPTION
**Pull request by moralejo**
Removed the duplicated masks for the DL1 parameters and DL1 images tables,
since they have one-to-one corresponding rows. Besides, code did not work
because images table now lacks event_id

### CODE REVIEW

**Comment by MaxNoe** [comment]
>  images table now lacks event_id

That should not be, everything needs to be identifiable

**Comment by moralejo** [comment]
> > images table now lacks event_id
> 
> That should not be, everything needs to be identifiable

I thought it was by design, since the rows in the two tables correspond one to one, so the event_id in both would be redundant. As long as things work properly, of course! If we want absolute safety we could put back that event_id in ctapipe...

**Comment by MaxNoe** [comment]
>  If we want absolute safety we could put back that event_id in ctapipe...

This is clearly a bug, since the tool has the option to only write the image table

**Comment by moralejo** [comment]
> 
> But wait: you are talking about the lstchain dl1 processing, not the ctapipe tool? Then this has nothing to do with ctapipe, it needs to be fixed here!

This happened as a result of a change in ctapipe, but indeed it makes no sense to change anything in ctapipe (or even in lstchain) since we will move as soon as possible to the new DL1. So no need to do anything about this.

File: lstchain/datachecks/containers.py [hunk 1]
```diff
@@ -129,9 +129,9 @@ def fill_event_wise_info(self, subrun_index, table, mask, geom,
         n_jump = 1+int(self.num_events/n_samples)
         # keep some info every n-jump-th event:
         sampled_event_ids = np.array(table['event_id'][mask][0::n_jump])
-        tib_time = np.array(table['tib_time'][mask][0::n_jump])
-        ucts_time = np.array(table['ucts_time'][mask][0::n_jump])
-        dragon_time = np.array(table['dragon_time'][mask][0::n_jump])
+        tib_time = np.array(table['tib_time'][mask][0::n_jump])*u.s
```
Comment (MaxNoe) [inline lstchain/datachecks/containers.py:132]: Do not use `array * u.<unit>`, it is much slower than `u.Quantity(somearray, u.unit, copy=False)`

**Comment by MaxNoe** [comment]
> This happened as a result of a change in ctapipe,

The containers don't duplicate this information anymore, this is right. For writing it out, we use the `EventIndex` and `TelescopeEventIndex` containers.

File: lstchain/datachecks/containers.py (hunk 1 above)
Comment (moralejo) [reply lstchain/datachecks/containers.py:132]: ok!

**Pull request merged**